### PR TITLE
Tribute

### PR DESCRIPTION
**Pull request by lhofmann**
Add documentation for the tribute example

### CODE REVIEW

**Comment by julien-tierny** [comment]
once your PR is merged, thanks for including a link to this example in the doxygen documentation of the involved modules.
see this commit for instance:
topology-tool-kit/ttk@2b9fd4c

File: python/tribute.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+calculator1.ResultArrayName = 'originalData'
+calculator1.Function = 'sqrt(PNGImage_X*PNGImage_X+PNGImage_Y*PNGImage_Y)'
+
+# create a new 'Tetrahedralize'
+tetrahedralize1 = Tetrahedralize(Input=calculator1)
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=tetrahedralize1)
+tTKPersistenceDiagram1.ScalarField = ['POINTS', 'originalData']
+tTKPersistenceDiagram1.InputOffsetField = [None, '']
```
Comment (julien-tierny) [inline python/tribute.py]: I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.

File: python/tribute.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# create a new 'Threshold'
+persistenceThreshold = Threshold(Input=appendDatasets1)
+persistenceThreshold.Scalars = ['CELLS', 'Persistence']
+persistenceThreshold.LowerThreshold = 8.5
+persistenceThreshold.UpperThreshold = 102.106426713382
+
+# create a new 'TTK TopologicalSimplification'
+tTKTopologicalSimplification1 = TTKTopologicalSimplification(Domain=tetrahedralize1, Constraints=persistenceThreshold)
+tTKTopologicalSimplification1.ScalarField = ['POINTS', 'originalData']
+tTKTopologicalSimplification1.InputOffsetField = [None, '']
```
Comment (julien-tierny) [inline python/tribute.py]: I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.

File: python/tribute.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+tTKTopologicalSimplification1.VertexIdentifierField = [None, '']
```
Comment (julien-tierny) [inline python/tribute.py]: I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.

File: python/tribute.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+# create a new 'TTK MorseSmaleComplex'
+tTKMorseSmaleComplex2 = TTKMorseSmaleComplex(Input=tTKTopologicalSimplification1)
+tTKMorseSmaleComplex2.ScalarField = ['POINTS', 'originalData']
+tTKMorseSmaleComplex2.OffsetField = ['POINTS', 'OutputOffsetScalarField']
```
Comment (julien-tierny) [inline python/tribute.py]: I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.

**Review by julien-tierny — CHANGES_REQUESTED** [review]
wonderful, just a couple tiny remarks.

**Review by julien-tierny — APPROVED** [review]

File: python/tribute.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+tetrahedralize1 = Tetrahedralize(Input=calculator1)
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=tetrahedralize1)
+tTKPersistenceDiagram1.ScalarField = ['POINTS', 'originalData']
+
+# create a new 'Threshold'
+threshold1 = Threshold(Input=tTKPersistenceDiagram1)
+threshold1.Scalars = ['CELLS', 'PairIdentifier']
+threshold1.LowerThreshold = -0.1
```
Comment (julien-tierny) [inline python/tribute.py]: I'd suggest to stick to the legacy format for the time being:
```
threshold1.ThesholdRange = [-0.1, 3560.0]
```

First, ParaView 5.10 is still a release candidate. 
Second, all TTK binary packages have been prepared against ParaView 5.9.

Once ParaView 5.10 is officially out and that we switch TTK's support to 5.10 officially too, we'll make a global pass on the examples to update the thresholds with a sed based script.

**Review by julien-tierny — CHANGES_REQUESTED** [review]
that looks awesome.
thanks for addressing my remark regarding ParaView 5.10's threshold API.

File: python/tribute.py (hunk 5 above)
Comment (lhofmann) [reply python/tribute.py]: Fixed

**Comment by julien-tierny** [comment]
alright, looks good!

**Comment by julien-tierny** [comment]
Now, could you please make a PR to ttk's source code to include this example in the doxygen and paraview documentation of the involved modules?
see this commit for instance:
topology-tool-kit/ttk@2b9fd4c

**Pull request merged**